### DISCUSSION TITLE
B2B-Features like price lists and tier pricing?

### DISCUSSION THREAD

**Discussion by trnc-ck** [vendurehq/vendure#2159] [Q&A]
hey,

what's the best way to implement customer group based pricing and tier pricing? so if customers get a special price for some products and pay less the higher the quantity is? Any experiences?

thanks

**Comment by michaelbromley** [comment]
Hi,

The mechanism to achieve this is the [OrderItemPriceCalculationStrategy](https://www.vendure.io/docs/typescript-api/orders/order-item-price-calculation-strategy/). Note that in v1.x, it's not really possible to do tiered pricing using this API because it does not know about the quantity of a given item in the order. That is fixed in the upcoming v2.
